### PR TITLE
fix: Properly display note path

### PR DESCRIPTION
**Pull request by acezard**
Related to https://trello.com/c/wpOZ52TO/95-%F0%9F%93%9D-notes-ne-pas-afficher-le-chemin-complet-dune-note-partag%C3%A9e

Using useFileWithPath() hook to get huiman readable filepath name

### CODE REVIEW

File: src/components/header_menu.jsx [hunk 1]
```diff
@@ -14,23 +14,23 @@ import { getDriveLink } from 'lib/utils'
...
+import useFileWithPath from 'hooks/useFileWithPath'
 
 const HeaderMenu = ({
   homeHref,
   backFromEditing,
   editorCorner,
   isPublic,
   file,
-  client,
+  cozyClient,
```
Comment (Crash--) [inline src/components/header_menu.jsx:25]: to me you should not pass the client in props. You should use `useClient()` inside the component to get it

Comment (acezard) [reply src/components/header_menu.jsx]: updated

File: src/components/header_menu.jsx [hunk 2]
```diff
@@ -61,13 +61,15 @@ const HeaderMenu = ({
...
-          </WithBreakpoints>
+          {fileWithPath?.path && (
+            <WithBreakpoints hideOn={Breakpoints.Mobile}>
+              <NotePath
+                drivePath={drivePath}
+                path={fileWithPath.path}
+                target="_blank"
+              />
+            </WithBreakpoints>
+          )}
```
Comment (Crash--) [inline src/components/header_menu.jsx:72]: I don't know if this is the right fix. 

What are you displaying if you share by link a note?
What are you displaying if you access a not inside a sub sub shared folder?

**Comment by acezard** [comment]
use le hook getpathname dans un nouveau component si is not public

**Review by Crash-- — APPROVED** [review]

**Pull request merged**